src/core/lua/luabfb.cpp

```diff
@@ -67,7 +67,6 @@ CLuaBFB::CLuaBFB(CStringDictionary::TStringId paInstanceNameId, const CLuaBFBTyp
 
 bool CLuaBFB::initialize() {
   //before calling super we need to configure the interface of the FB
```

Comment:
Please also remove the obsolete comment.

Reply:
removed the entire function, as suggested above

Reply:
Done